firebase.json

```diff
@@ -39,7 +39,7 @@
       { "source": "/desktop/windows", "destination": "/development/platform-integration/windows/building", "type": 301 },
       { "source": "/desktop/macos", "destination": "/development/platform-integration/macos/building", "type": 301 },
       { "source": "/desktop/linux", "destination": "/development/platform-integration/linux/building", "type": 301 },
-      { "source": "/docs/perf/rendering/*", "destination": "/docs/perf", "type": 301 },
+      { "source": "/perf/rendering/*", "destination": "/perf", "type": 301 },
```

Comment:
This will actually fails the build, the page will be moved to /docs/perf so every links targeting this will be 404 during links check.

Reply:
The upstream doesn't has the same issue because it uses `docs.flutter.dev`, and we use `flutter.cn/docs`.

Reply:
None of the other redirects have `/docs`. I think the redirect was just hiding the broken link in the header.